src/Integrals.jl

```diff
@@ -292,11 +292,6 @@ function ChainRulesCore.rrule(::typeof(__solvebp), prob, alg, sensealg, lb, ub,
     out, quadrature_adjoint
 end
 
-ZygoteRules.@adjoint function ZygoteRules.literal_getproperty(sol::SciMLBase.IntegralSolution,
-                                                              ::Val{:u})
-    sol.u, Δ -> (SciMLBase.build_solution(sol.prob, sol.alg, Δ, sol.resid),)
-end
```

Comment:
I don't see where the chain rule is for this?

Reply:
Zygote seems to be broken in more than one way, so I'll convert this to draft until I have it all figured out.